packages/material-ui/src/FormControl/FormControl.js

```diff
@@ -86,12 +87,16 @@ class FormControl extends React.Component {
         error,
         filled,
         focused,
+        hover,
         margin,
         onBlur: this.handleBlur,
         onEmpty: this.handleClean,
         onFilled: this.handleDirty,
         onFocus: this.handleFocus,
+        onMouseEnter: this.handleMouseEnter,
```

Comment:
Does the event need to be attached to the `input` or can we use event bubbling to attach it to the `FormControl`?